edb/edgeql/codegen.py

```diff
@@ -971,7 +971,11 @@ def visit_AlterDropInherit(self, node: qlast.AlterDropInherit) -> None:
             self.visit_list(node.bases)
 
     def visit_CreateDatabase(self, node: qlast.CreateDatabase) -> None:
-        self._visit_CreateObject(node, 'DATABASE')
+        def after_name() -> None:
```

Comment:
This isn't relevant to this PR, I think.

Reply:
Right, but it's in general trying to account for `CREATE DATABASE foo FROM bar` syntax. This exists in the grammar and seems to be use din some testing (https://github.com/edgedb/edgedb/blob/29cec52cf3609389503ed21ef2911657f5f994a6/edb/testbase/server.py#L946-L947).

I can wrap that into a separate tiny codegen/syntax PR.